seedfarmer/resources/projectpolicy.yaml

```diff
@@ -54,23 +54,23 @@ Resources:
               - logs:GetLogGroupFields
               - logs:GetQueryResults
             Resource:
-              - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/codeseeder-${ProjectName}*"
+              - Fn::Sub: "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/codeseeder-${ProjectName}*"
```

Comment:
Why these changes?

Reply:
`Fn::Sub` is consistent with our bootstrap templates and can be processed by any yaml parser. Just to not have to bring `cfn-flip` as a dependency.